src/Shared/Common/Interfaces/IBlobstorageHelper.cs

```diff
@@ -0,0 +1,10 @@
+namespace NHS.ServiceInsights.Common;
+
+using NHS.ServiceInsights.Model;
```

Comment:
Can you move the using directives to the top of the file for consistency, please